app/components/atoms/CarouselCircle/CarouselCircle.styles.tsx

```diff
@@ -0,0 +1,24 @@
+import styled, {css} from 'styled-components';
+
+export interface CarouselCircleProps {
+  active?: boolean;
+}
+
+const activeStyles = css`
+  background-color: ${({theme}) => theme.colors.backgroundDarker};
+  border-width: 3px;
+  border-color: ${({theme}) => theme.colors.black};
+  border-style: solid;
+`;
+
+export const Bullet = styled.span<CarouselCircleProps>`
```

Comment:
It's called "CarouselCircle" in the storybook but "Bullet" in the implementation. Debugging this will be hell. Please reduce the naming variation.